packages/manager/apps/vrack-services/src/components/UpdateDisplayNameMessages.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+import { OsdsMessage, OsdsText } from '@ovhcloud/ods-components/react';
+import { MutationState, useMutationState } from '@tanstack/react-query';
+import {
+  ODS_MESSAGE_TYPE,
+  ODS_TEXT_LEVEL,
+  ODS_TEXT_SIZE,
+} from '@ovhcloud/ods-components';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import { useTranslation } from 'react-i18next';
+import {
+  updateVrackServicesNameQueryKey,
+  UpdateVrackServicesNameMutationParams,
+} from '@/api';
+
+export const UpdateDisplayNameMessage: React.FC = () => {
+  const { t } = useTranslation('vrack-services/listing');
+  const updateDisplayNameMutations = useMutationState<
+    MutationState<
+      unknown,
+      Error,
+      UpdateVrackServicesNameMutationParams,
+      unknown
+    >
+  >({
+    filters: {
+      mutationKey: updateVrackServicesNameQueryKey(),
+      exact: true,
+    },
+  });
+
+  const [hiddenMessage, setHiddenMessage] = useState([]);
+
+  if (!updateDisplayNameMutations.length) {
+    return null;
+  }
+
+  return (
+    <>
+      {updateDisplayNameMutations.map(
+        (
+          updateMutation: MutationState<
+            unknown,
+            Error,
+            UpdateVrackServicesNameMutationParams,
+            unknown
+          >,
+          mutationIndex: number,
+        ) =>
+          updateMutation.status === 'success' &&
+          !hiddenMessage.includes(updateMutation.submittedAt) && (
```

Comment:
if we don't need the mutationIndex anymore, we can maybe simplify the `.map` by using

```
.filter(updateMutation => updateMutation.status === 'success' &&
          !hiddenMessage.includes(updateMutation.submittedAt))
.map(updateMutation => (<OsdsMessage ... />)
```